pytripgui/canvas_vc/plotter/images/ctx_image.py

```diff
@@ -11,9 +11,28 @@ def __init__(self, axes):
         self.zorder = 1
 
     def plot(self, data: Ctx) -> None:
+        extent = self.calculate_extent(data)
         self._image = self._axes.imshow(data.data_to_plot,
                                         cmap=self._colormap,
                                         vmin=data.contrast_ct[0],
                                         vmax=data.contrast_ct[1],
                                         aspect=data.aspect,
+                                        extent=extent,
+                                        origin='lower',
                                         zorder=self.zorder)
+        self._set_labels(data)
+
+    def _set_labels(self, data: Ctx):

```

Comment:
Add a comment explaining why labels are set only on Ctx and not on a Dos and LET (cause the other simulation results are just overlays, right?).

Reply:
Done by changing the place where method is defined and invoked